src/backend/distributed/utils/citus_safe_lib.c

```diff
@@ -165,6 +167,47 @@ SafeStringToUint64(const char *str)
 }
 
 
+/*
+ * SafeStringToFloat converts a string containing a number to a float. When it
+ * fails it calls ereport.
+ *
+ * The different error cases are inspired by
+ * https://stackoverflow.com/a/26083517/2570866
+ */
+float
+SafeStringToFloat(const char *str)
```

Comment:
unused in this diff, left over from earlier itterations?

Reply:
removed it